src/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh

```diff
@@ -348,14 +343,11 @@ if [[ $RUN_TRANSFER -eq 1 ]]; then
       fi
 
       transfer_queue=dev_transfer
-
       jobname=transfer_${RADMON_SUFFIX}
-      job="${IG_SCRIPTS}/Transfer.sh --nosrc ${RADMON_SUFFIX}"
-
       export WEBDIR=${WEBDIR}/${RADMON_SUFFIX}/pngs
 
       cmdfile="${PLOT_WORK_DIR}/transfer_cmd"
-      echo "${IG_SCRIPTS}/Transfer.sh --nosrc ${RADMON_SUFFIX}" >$cmdfile
+      echo "${IG_SCRIPTS}/transfer.sh" >$cmdfile
       chmod 755 $cmdfile
 
       run_time="$rhr$cmin"	# HHMM format for qsub
```

Comment:
So should transfer.sh be run from all systems, then, or just wcoss2?

Reply:
Just wcoss2.  There's a preexisting check at line 327 (not in view) for that.

Reply:
Ah, got it.